### DISCUSSION TITLE
Nextcloud port detection tool fails but online tool succeeds

### DISCUSSION THREAD

**Discussion by clareclever** [nextcloud/all-in-one#1264] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Detailed issue
I followed the AIO steps to setup the Nextcloud docker container. I had done the initial domain check but it failed to access my domain on port 443. I used the port checking tool as stated in the notification and it said that port was up on my IP so I added the command to skip the domain check. I have port 443 open on my firewall and have it set to forward on my router. Is there anything I should try or look into to determine why I can access my AIO instance via the local address (10.0.0.128:8080) but not my public DNS address? I saw something about having to configure your DNS, I don’t think I did that… what does that consist of?

### Host OS 
Ubuntu Server 22.04

#### Nextcloud AIO version 
I cannot access my Nextcloud interface to determine this information.

**Comment by szaimen** [comment, marked as answer]
Hi, 

what do you see when you open the AIO interface? Can you post a screenshot?

Did you already start the containers from the AIO interface?

**Reply by clareclever** [reply]
Oh, thanks for the reply. I actually got it to work. When I connected to it through my DNS I needed to connect to it on a particular port first to set it up then go to the main base URL. I got it working now! Thanks for all the help.
